CMakeLists.txt

```diff
@@ -155,7 +165,7 @@ if (MINGW OR APPLE)
       SET(Mscore_SHARE_NAME    "mscore.app/")
   endif(MINGW)
 else (MINGW OR APPLE)
-      SET(Mscore_INSTALL_NAME  "mscore-${MUSESCORE_VERSION}/")
+      SET(Mscore_INSTALL_NAME  "mscore${CMAKE_INSTALL_SUFFIX}-${MUSESCORE_VERSION}/")
```

Comment:
CMAKE_INSTAL_SUFFIX is not a Cmake variable. It would be better to keep the CMAKE_ namespace for official CMake variable.
No idea if it would solve the particular issue here but CMake provides http://www.cmake.org/cmake/help/v3.0/variable/CMAKE_EXECUTABLE_SUFFIX.html